README.md

```diff
@@ -1,2 +1,57 @@
 # hbase-rule
-JUnit rule which provides an embedded HBase server.
+JUnit rule which provides an embedded HBase server. It can be setup by custom configuration or default. The rule has also helper methods to work with hbase table and namespace such as create, delete, truncate and etc.
+
+# Sample Usage
+
```

Comment:
Try to have more simple and narrative approach. I can suggest these changes:
- Remove these static constants.
- At first have a simple sample without setup() and tearDown(). Just create the table using the builder of the rule class. and implement a simple test. (Also remove the static constants)
- Then in a new paragraph, tell that it is possible to define the table on setup or delete the data on tearDown and write down the setup() and tearDown() sample codes.
- Then in a new paragraph tell that the rule provides several other methods such as countRows(), deleteTable(), trucateTable() and write down some samples.

You should also define another section: "Add it to your project". That is like the ones we have on other github projects but also should contain more notes: the dependencies defined in the pom of this project. The user should define them with its appropriate choice of version it its pom. He should define the server and minicluster dependencies in test scope.
-